hw/ip/flash_ctrl/data/flash_ctrl.hjson.tpl

```diff
@@ -1119,9 +1129,9 @@
             '''
           },
           { bits: "1",
-            name: "phy_intg_err",
+            name: "phy_prog_intg_err",
             desc: '''
-              The flash memory encountered an integrity error on the host access interface.
+              The flash controller encountered a program data transmission integrity error.
```

Comment:
So this is different from `status.prog_intg_err`? It's looking at the integrity of the data between... the controller and the phy? Is that right?

Reply:
gah! i forgot to remove this one. This is all prog_intg_err.

And it should only exist in ERR_CODE and FAULT_STATUS.
The former is when software encounters it as part of a flash controller initiated program.
The latter is when the life cycle management interface encounters it as part of its operations.

The one in STD_FAULT_STATUS should not exist anymore. 
I also messed up the language in the transmission integrity section because of this. I will fix.